tests/core/DocumentComposer.spec.ts

```diff
@@ -47,19 +47,19 @@ describe('DocumentComposer', async () => {
         verificationMethod: [
           {
             id: '#anySigningKey',
-            controller: '',
+            controller: did.shortForm,
```

Comment:
If we use long form controller as mentioned in the comment threads above, will need a new test for composing long form. I think simply copying the format of the `transformToExternalDocument` test and call `DocumentComposer.transformToExternalDocument(didState, did, published);` with published set to false should do the trick.